www/include/views/graphs/exportData/ExportCSVServiceData.php

```diff
@@ -135,20 +135,20 @@ function get_error($str)
     $stmt->bindValue(':index', $index, \PDO::PARAM_INT);
     $stmt->execute();
 
-    while ($index_data = $stmt->fetch(\PDO::FETCH_ASSOC)) {
-        $listMetric[$index_data['metric_id']] = $index_data['metric_name'];
-        $listEmptyMetric[$index_data['metric_id']] = '';
+    while ($indexdata = $stmt->fetch(\PDO::FETCH_ASSOC)) {
+        $listMetric[$indexdata['metric_id']] = $indexindexdatadata['metric_name'];
```

Comment:
```suggestion
        $listMetric[$indexData['metric_id']] = $indexData['metric_name'];
```

Reply:
please use CamelCase and check the indexindexdatadata variable ;)

Reply:
Yep sorry about that this is outdated now since last commit :+1: fast copy/paste gone wrongly :rofl: